worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -349,12 +362,15 @@ class PushDataHandler(val workerSource: WorkerSource) extends BaseMessageHandler
       // will fast stop pushing data to the worker, we won't return congest status. But
       // in the long term, especially if this issue could frequently happen, we may need to return
       // congest&softSplit status together
+      writeLocalData(Seq(fileWriter), body, shuffleKey, None, None)
```

Comment:
I think we still need to get the status of `writeLocalData` and call `onFailure` if not `Success`.